base/docs/helpdb.jl

```diff
@@ -8590,9 +8590,9 @@ As a special case, if `x` is an `AbstractString` (for textual MIME types) or a `
 reprmime
 
 doc"""
-    rm(path::AbstractString; recursive=false)
+    rm(path::AbstractString; force=false, recursive=false)
```

Comment:
rst docs need updating for this too. make signature consistent then run genstdlib.jl